### ISSUE TITLE
Update some older math routines

### ISSUE THREAD

**Issue by cacraigucar** [ESCOMP/CAM#986]
### Issue Type

Code Clean-up

### Issue Description

A number of older math routines need updated code

A spreadsheet with the targeted codes will be linked in here when it is ready

@do

### Will this change answers?

I Don't Know

### Will you be implementing this yourself?

Any CAM SE can do this

**Comment by cacraigucar** [comment]
@briandobbins - This may introduce roundoff level changes.  Should it be done soon?

**Comment by briandobbins** [comment]
Yes, but it's still secondary to some of the science changes & tunings -- I gave Dave a heads-up on this a while back (eg, that we'd need time after the tuning is set), and will remind him again.  Roundoff-level changes _shouldn't_ impact the tuning.
